generated/json_data/qol.jsonc

```diff
@@ -6805,8 +6881,39 @@
                         }
                     ]
                 },
-                "Phendrana Shorelines": {
-                    "addConnections": [
+                "Phendrana Shorelines": {/* Unused ice breaking particles, present in Remastered.
```

Comment:
I don't like the idea of having these large blocks of commented-out changes. It's a slippery slope to a messy file. If you are working on a feature that isn't fully finished/tested, keep it out of the pull request entirely. I suggest making separate git branches for separate features so that you can submit/test them independently from one another.